README.adoc

```diff
@@ -216,36 +200,30 @@ First, create a `src/test/java/it/io/openliberty/guides/metrics/MetricsTest.java
-the `200` response code, which is returned with the `connectToEndpoint()` private method. Next, the
-test case makes a connection to the `https://localhost:9443/metrics/application` URL to get the
-content of the metrics as plain text. The test case then uses the `validateMetric()` method to
-assert whether the time that is needed to get the system properties for `localhost` is less than 4
-seconds.
-
-* The `testListCount()` test case is used to test the `@Counted` annotation. This test case uses the
-`connectToEndpoint()` method to send a request to the `http://localhost:9080/inventory/systems` URL.
-Next, the test case uses the `validateMetric()` method to assert whether the `list()` method in the
-`InventoryManager` class was invoked once.
-
-* The `testInventorySize()` test case is used to test the `@Gauge` annotation. This test case uses the
-`connectToEndpoint()` method to send a request to the `http://localhost:9080/inventory/systems/localhost`
-URL. Because of the request, the `getTotal()` method in the `InventoryManager` class returns a value of one,
-which represents the localhost. In another words, the inventory size increases by one due to the request.
-Then the `validateMetric()` method asserts if the inventory size is equal to one.
+* The `testGetPropertiesTime()` test case validates the `@Timed` metric. It sends a request to the 
+`http://localhost:9080/inventory/systems/localhost` URL to access the `inventory` service and add
+the `localhost` host to the inventory. Next, the test case makes a connection to the 
+`https://localhost:9443/metrics/application` URL to retrieve application metrics as plain text.
+It then uses the `validateMetric()` method to assert whether the time that is needed to retrieve
+the system properties for `localhost` is less than 4 seconds.
+
+* The `testListCount()` test case validates the `@Counted` metric. The test case sends a request to the 
+`http://localhost:9080/inventory/systems` URL to retrieve the whole inventory and then, it asserts
+that this endpoint is only accessed once.
+
+* The `testInventorySize()` test case validates the `@Gauge` metric. The test case first ensures
+that the  `localhost` host is in the inventory. It then looks for the `@Gauge` metric and asserts
+that the inventory size is equal to one.
+
+The `oneTimeSetup()` method retrieves the port number for the server and builds a base URL string
+to setup the tests. Apply the `@BeforeClass` annotation to this method to execute it before any of
+the test cases.
+
+The `setup()` method creates a JAX-RS client that makes HTTP requests to the `inventory` service.
+Register this client with a `JsrJsonpProvider` JSON-P provider to process JSON resources. The
+`teardown()` method destroys this client instance. Apply the `@Before` and `@After` annotations to
+these methods respectively so that they execute before and after every test case. Apply these annotations
```

Comment:
Our grammar checker isn't a fan of the word "respectively." What about, "Apply the `@Before` annotation so that a method executes before other test cases, and apply an `@After` annotation so that a method executes after other test cases."